src/Entity/Jecoute/News.php

```diff
@@ -45,12 +52,19 @@
  *                     },
  *                 }
  *             }
- *         }
+ *         },
+ *         "get_private": {
+ *             "path": "/v3/jecoute/news",
+ *             "method": "GET",
+ *             "normalization_context": {"groups": {"jecoute_news_read_dc"}},
+ *             "access_control": "is_granted('IS_FEATURE_GRANTED', 'news')",
+ *         },
  *     },
  *     itemOperations={
- *         "get": {
+ *         "get_public": {
```

Comment:
could you add the security check for this route, please? 
`"access_control": "is_granted('ROLE_OAUTH_SCOPE_JEMARCHE_APP')",`